src/modules/blog/blog.controller.ts

```diff
@@ -7,11 +7,23 @@ import { BadRequestException, NotFoundException } from "../../shared/exceptions/
 import { Blog } from './blog.entity';
 import { User } from '../user/user.entity';
 import { BlogService } from './blog.service';
+import { CommentService } from '../comments/comments.service';
 import { validateRequestBody } from '../../shared/validators/request-body.validator';
 import { validateRequestParams } from '../../shared/validators/request-params.validator';
+import { commentCreateDtoSchema, CreateCommentDto } from "../comments/dto/create-comment.dto";
+import { Comment } from "../comments/comments.entity";
 
 export const blogController = Router();
-const blogService = new BlogService(db.connection.getRepository(Blog), db.connection.getRepository(User))
+
+const blogService = new BlogService(
+  db.connection.getRepository(Blog),
+  db.connection.getRepository(User)
+)
+const commentService = new CommentService(
+  db.connection.getRepository(Comment),
+  db.connection.getRepository(User),
+  db.connection.getRepository(Blog)
+)
 
 blogController.post(
```

Comment:
Is your routes are protected ?
In 2 task we implemented authentication. You should have create, put delete post protected with jwt token validation

Reply:
You talk about Auth Guard?
If am I right, im already did it, in the next PR I'm planning to implement 3 guards (Auth Guard, User-Self Guard, Roles Guard)

I will send you link to it after this